packages/tsconfig.json

```
@@ -10,6 +10,7 @@
 		{ "path": "./components" },
 		{ "path": "./composite-checkout" },
 		{ "path": "./create-calypso-config" },
+		{ "path": "./create-selector" },
```

We should update that to reflect the new package name.

```suggestion
		{ "path": "./state-utils" },
```